tmt/base.py

```diff
@@ -1968,7 +1969,7 @@ def __init__(self,
                  *,
                  path: Path = Path.cwd(),
                  tree: Optional[fmf.Tree] = None,
-                 context: Optional[tmt.utils.FmfContextType] = None,
+                 fmf_context: Optional[tmt.utils.FmfContextType] = None,
```

Comment:
Looks to me that this change broke the `/tests/integration/the_rest`
```
>       tree_f36_intel = tmt.Tree(
            logger=tmt.log.Logger.create(),
            path='.',
            context={
                'distro': ['fedora-36'],
                'arch': ['x86_64']})
E       TypeError: Tree.__init__() got an unexpected keyword argument 'context'
```

So in the `tests/integration/test_nitrate.py` should be change the creating of `tmt.Tree` with `context` -> `fmf_context`.

Reply:
Correct, fixed in 8884f543.